app/controllers/operationsAndFunds/StartBankDetailsController.scala

```diff
@@ -33,8 +37,12 @@ class StartBankDetailsController @Inject()(
     view: StartBankDetailsView
    )(implicit appConfig: FrontendAppConfig) extends LocalBaseController {
 
-  def onPageLoad(): Action[AnyContent] = (identify andThen getData andThen requireData) { implicit request =>
-
-    Ok(view())
+  def onPageLoad(): Action[AnyContent] = (identify andThen getData andThen requireData).async { implicit request =>
+    if (request.userAnswers.get(Section1Page).contains(false)) {
```

Comment:
redirecting to service reset page if SectionPage1 is not defined... Please fix this